### ISSUE TITLE
Add validation for Conference timezone

### ISSUE THREAD

**Issue by Ana06** [openSUSE/osem#2420]
If the time zone is nil when creating a conference it will fail the database level. We should have a validation as well.

To reproduce it, execute in the Rails console:

```
Conference.create(organization_id: 1, title: "conference", short_title: "conf", start_date: DateTime.now + 7.days, end_date: DateTime.now + 10.days)
```

which raises:

```
ActiveRecord::StatementInvalid (PG::NotNullViolation: ERROR:  null value in column "timezone" violates not-null constraint
```

**Comment by ibrahimelbanna** [comment]
can i work on it

**Comment by Ana06** [comment]
sure! go ahead @ibrahimelbanna

**Comment by ibrahimelbanna** [comment]
#2441 @Ana06

**Comment by Ana06** [comment]
Fixed in #2441
